app/components/steps/step-1/step-1.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'step-1',
+  templateUrl: './step-1.component.html'
+})
+export class Step1Component {
```

Comment:
Wasn't sure whether to keep it for later use and whether it would be helpful to have. I can remove it completely, since it doesn't appear necessary.